src/components/Settings/GameBoardSettings.vue

```diff
@@ -27,6 +29,55 @@
       </select>
     </div>
 
+    <div class="setting">
+      <i class="fas fa-bars" />
+      <h5>Header position</h5>
+
+      <select v-model="value[platform.code].position" @change="$emit('save')">
+        <option
+          v-for="{ id, name } in positions"
+          :key="id"
+          :value="id"
+        >
+          {{ name }}
+        </option>
+      </select>
+    </div>
+
+    <div class="setting">
+      <i class="fas fa-shapes" />
+      <h5>Border Radius</h5>
+
+      <toggle-switch
+        id="borderRadius"
+        @change="$emit('save')"
+        v-model="value[platform.code].borderRadius"
+      />
+    </div>
+
+    <div class="setting">
+      <i class="fas fa-stream" />
+      <h5>Hide list backgrounds</h5>
+
+      <toggle-switch
+        id="hideListBackgrounds"
+        @change="$emit('save')"
+        v-model="value[platform.code].hideListBackgrounds"
+      />
+    </div>
+
+    <h3>Gameboard</h3>
+    <div class="setting">
+      <i class="fas fa-heading" />
+      <h5>Hide amount of games next to list title</h5>
+
+      <toggle-switch
+        id="hideGameAmount"
+        @change="$emit('save')"
+        v-model="value[platform.code].hideGameAmount"
+      />
+    </div>
```

Comment:
I like this, maybe we can hide game count by default to achieve a cleaner look and let users decide if they want to show it?

If you go that route, you could change the setting name to `showGameCount` or similar

Reply:
Sure, I'd like that as a default, too.